base/gmp.jl

```diff
@@ -69,6 +69,93 @@ function __init__()
     end
 end
 
+
+module mpz
```

Comment:
would rather stick with the established naming conventions. I don't think it's especially necessary to have julia wrappers for C functions use exactly the same names as in the C library, when the conventions or meanings for names differ in idiomatic Julia